extensions/sql-database-projects/src/dialogs/deployDatabaseQuickpick.ts

```diff
@@ -70,6 +70,35 @@ export async function launchDeployAppIntegrationQuickpick(project: Project): Pro
 	};
 }
 
+async function launchEulaQuickPick(baseImage: string): Promise<boolean> {
+	const baseImages = uiUtils.getDockerBaseImages();
+	const imageInfo = baseImages.find(x => x.name === baseImage);
+	let result: string | undefined = constants.openEulaString;
+	if (imageInfo?.agreementInfo?.link) {
+		const link = imageInfo?.agreementInfo?.link;
+
+		// Ask user to update app settings or not
+		//
+		let choices = [
+			constants.yesString,
+			constants.noString,
+			constants.openEulaString
+		];
+		let options: vscode.QuickPickOptions = {
+			placeHolder: `${uiUtils.getAgreementDisplayText(imageInfo?.agreementInfo)}`,
```

Comment:
You already know this is truthy so you can just do `imageInfo.agreementInfo` here